### PR TITLE
Timezones not showing bug

### PR DESCRIPTION
**Pull request by abinashpa**
## Notion card
[Notion Link](https://www.notion.so/saeloun/No-timezones-on-the-new-company-page-0b76b7d20c674fdf90b32bf802dd8e41)
## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to
      not work as expected)
- [ ] This change requires a documentation update

### Checklist:

- [x] I have manually tested all workflows
- [ ] I have performed a self-review of my own code
- [ ] I have added automated tests for my code

### CODE REVIEW

**Comment by apoorv1316** [comment]
@abinashpa Is it ready for review?

**Comment by abinashpa** [comment]
> @abinashpa Is it ready for review?

Yes

**Review by apoorv1316 — CHANGES_REQUESTED** [review]
@abinashpa 'company/new' page is broken. Please check it. 
Screenshot 2022-08-04 at 10 21 18 AM

**Comment by abinashpa** [comment]
Screenshot 2022-08-09 at 11 05 19 AM
Working fine.

**Comment by supriya3105** [comment]
@abinashpa Please rebase with develop as the test that is failing with this PR has been been fixed by Gowsik earlier.

**Review by apoorv1316 — APPROVED** [review]

**Review by gowsik-ragunath — APPROVED** [review]
LGTM!

**Comment by gowsik-ragunath** [comment]
@akhilgkrishnan It seems we are adding alot of JS code in action view can we integrate stimulus js in the app?

File: app/assets/js/org_timezones.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+/* eslint-disable space-before-function-paren */
+/* eslint-disable func-style */
+
+document.addEventListener("DOMContentLoaded", function () {
```
Comment (akhilgkrishnan) [inline app/assets/js/org_timezones.js:4]: @abinashpa cant we move this to pack folder instead of assets?

**Review by akhilgkrishnan — APPROVED** [review]
LGTM

**Comment by supriya3105** [comment]
@abinashpa Please fix the tests that are failing

**Pull request merged**